packages/platform-sdk/test/integration-tests/custom-object/custom-object.test.ts

```diff
@@ -0,0 +1,143 @@
+      .customObjects()
+      .withContainerAndKey({
+        container: responseCreatedCustomObject.body.container,
+        key: responseCreatedCustomObject.body.key,
+      })
+      .delete({
+        queryArgs: { version: responseCreatedCustomObject.body.version },
+      })
+      .execute()
+
+    expect(responseCustomObjectsDeleted.statusCode).toEqual(200)
+  })
+
+  it('should get a customObject by Container and Key ', async () => {
+    const customObject = await createCustomObject()
+
+    const getCustomObject = await apiRoot
+      .customObjects()
+      .withContainerAndKey({
+        container: customObject.body.container,
+        key: customObject.body.key,
+      })
+      .get()
+      .execute()
+
+    expect(getCustomObject.body.version).not.toBe(null)
+    expect(getCustomObject.body.id).toEqual(customObject.body.id)
+
+    await deleteCustomObject(
+      getCustomObject.body.container,
+      getCustomObject.body.key,
+      getCustomObject.body.version
+    )
+  })
+
+  it('should update a customObject with object', async () => {
+    const customObject = await createCustomObject()
+
+    const foo = {
+      1: "World's End",
+      2: 'Winchester',
+    }
+
+    const customObjectDraft: CustomObjectDraft = {
+      key: customObject.body.key,
+      container: customObject.body.container,
+      value: foo,
+    }
+
+    const updateCustomObject = await apiRoot
+      .customObjects()
+      .post({
+        body: customObjectDraft,
+      })
+      .execute()
+
+    expect(updateCustomObject).not.toBe(null)
+    expect(
+      updateCustomObject.body.value[
+        Object.keys(updateCustomObject.body.value)[0]
+      ]
+    ).toEqual("World's End")
+
+    await deleteCustomObject(
+      updateCustomObject.body.container,
+      updateCustomObject.body.key,
+      updateCustomObject.body.version
+    )
+  })
+
+  it('should update a customObject with new string value', async () => {
+    const customObject = await createCustomObject()
+
+    const customObjectDraft: CustomObjectDraft = {
+      key: customObject.body.key,
+      container: customObject.body.container,
+      value: 'foo',
+    }
+
+    const updateCustomObject = await apiRoot
+      .customObjects()
+      .post({
+        body: customObjectDraft,
+      })
+      .execute()
+
+    expect(updateCustomObject).not.toBe(null)
+    expect(updateCustomObject.body.value).toEqual('foo')
+
+    await deleteCustomObject(
+      updateCustomObject.body.container,
+      updateCustomObject.body.key,
+      updateCustomObject.body.version
+    )
+  })
+
+  it('should query a customObject', async () => {
+    const customObject = await createCustomObject()
+
+    const queryCustomObject = await apiRoot
+      .customObjects()
+      .withContainerAndKey({
+        container: customObject.body.container,
+        key: customObject.body.key,
+      })
+      .get({
+        queryArgs: {
+          where: 'id=' + '"' + customObject.body.id + '"',
+        },
```

Comment:
I think you should change this to another example, because it looks like query by ID like this is OK too and actually it is not. For custom object you could query by `createdAt`.

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/474/commits/30df75a296b20fa7b74e44263eed3e70bd141d0c